env/src/main/java/com/sap/cloud/security/config/cf/CFConstants.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * SPDX-FileCopyrightText: 2018-2021 SAP SE or an SAP affiliate company and Cloud Security Client Java contributors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.sap.cloud.security.config.cf;
+
+/**
+ * Constants that simplifies access to service configuration properties in the
+ * Cloud Foundry environment.
+ */
+public class CFConstants {
```

Comment:
why have you moved that?
We can not remove classes from java-api, only from java-security, as java-security depends on env.